awx/main/models/ha.py

```
@@ -485,6 +485,7 @@ def _get_related_jobs(self):
 
     class Meta:
         app_label = 'main'
+        ordering = ('pk',)
         permissions = [('use_instancegroup', 'Can use instance group in a preference list of a resource')]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)